bears/js/JSHintBear.py

```diff
@@ -35,7 +35,7 @@ class JSHintBear:
     """
 
     LANGUAGES = {'JavaScript'}
-    REQUIREMENTS = {NpmRequirement('jshint', '2')}
+    REQUIREMENTS = {NpmRequirement('jshint', '2.9.5')}
```

Comment:
Shouldn't we update it in `package.json` too?

Reply:
It is

Reply:
yeah my bad, didn't scroll fully. Sorry for the inconvenience

Reply:
Unnecessary change ;P